python/docs/migration/basics.md

```diff
@@ -1,52 +1,43 @@
-# v0 to v1 Basics
+# Basics - v0 to v1
 
-## Performance Improvement
```

Comment:
Someone with more context on the specifics of how performance was improved could provide me with a simple description, and I can update.